src/app/workshop-details/workshop-details.component.ts

```diff
@@ -70,6 +70,17 @@ export class WorkshopDetailsComponent {
         this.arrowKeyfunction();
     }
 
+    createExternalLink(url: string)
```

Comment:
can we create external link before hand and bind to a variable instead of binding to a function?